api/app/tests/apps/api/v1/test_private_signal_endpoint.py

```diff
@@ -1525,6 +1527,54 @@ def test_update_directing_departments_on_parent_signal(self):
 
         self.assertEqual(len(response.json()), 1)
 
+    def test_deadlines_available_via_api_detail_endpoint(self):
+        # self.signal has a category that has no ServiceLevelObjective associated
+        # with it, so deadlines should be None
+        result = self.client.get(self.detail_endpoint.format(pk=self.signal_no_image.id))
+        result_json = result.json()
+
+        deadline = result_json['category']['deadline']
+        deadline_factor_3 = result_json['category']['deadline_factor_3']
+        self.assertEqual(deadline, None)
+        self.assertEqual(deadline_factor_3, None)
```

Comment:
Do we want to add this to the JSON Schema?

Reply:
Will do